content/spire/try/getting-started-linux-macos-x.md

```diff
@@ -32,7 +32,7 @@ The commands in this getting started guide can be run as a standard user or root
 ## Downloading SPIRE for Linux
 
 ```
-$ curl -s -N -L https://github.com/spiffe/spire/releases/download/0.9.0/spire-0.9.0-linux-x86_64-glibc.tar.gz | tar xz
+$ curl -s -N -L https://github.com/spiffe/spire/releases/download/0.10.0/spire-0.10.0-linux-x86_64-glibc.tar.gz | tar xz
```

Comment:
Hi Dan, while we're updating this file, let's use Max C's new Hugo shortcodes, which I can understand you probably never heard of (invented a few days ago). They're described on the README at https://github.com/spiffe/spiffe.io under "Latest SPIRE version".

So the new curl command would be:
curl -s -N -L https://github.com/spiffe/spire/releases/download/{{< spire-latest "tag" >}}/{{< spire-latest "tarball" >}} | tar xz

FYI, this page on spiffe.io uses these shortcodes: https://spiffe.io/spire/docs/install-server/